release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+- [#2997](https://github.com/grafana/k6/pull/2997) Added a new exit code (`109`), used on a go panic.
+- [browser#788](https://github.com/grafana/xk6-browser/pull/788) Updated the browser readme to highlight that it is now a module in k6.
+- [browser#803](https://github.com/grafana/xk6-browser/pull/803) Users are now warned if the `browser.close` method is called more than once.
+- [browser#820](https://github.com/grafana/xk6-browser/pull/820) Added error handling to wildcard selectors, which cleans up the error output in the terminal.
+- [browser#843](https://github.com/grafana/xk6-browser/pull/843) Removed the build step from github actions. From this release onwards, no new standalone browser binaries will be built and available from the [releases](https://github.com/grafana/xk6-browser/releases) section. The latest version of the browser module will be available in the k6 binary which can be found in the [k6 releases](https://github.com/grafana/k6/releases) page.
+- [browser#848](https://github.com/grafana/xk6-browser/pull/848) Multiple k6 instances can now connect to one browser to run concurrent tests. This update empowers high-concurrency browser testing with multiple VUs and instances. Using the new `browserType.Connect` API, users can now connect to an existing browser instance and execute concurrent tests, which was not possible previously.
+
+## Bug fixes
+
+- [#2984](https://github.com/grafana/k6/pull/2984) Fixed wrongly assigned HTTP `POST` and `PUT` methods for the `tracing.instrumentHTTP`. Thanks, @marcin-maciej-seweryn!
+- [#2928](https://github.com/grafana/k6/pull/2928) Handled a new behavior of `filepath.Join` on windows with go1.20, which could cause issues for the `k6 archive` and `k6 cloud` commands.
+- [#2915](https://github.com/grafana/k6/pull/2915) Fixed `check` that could return incorrect values for some cases with many preallocated VUs.
+- [#2953](https://github.com/grafana/k6/pull/2953) Fixed active VU reporting by arrival-rate executors.
+- [#3006](https://github.com/grafana/k6/pull/3006) `xk6-websockets` updated to v0.2.0 which fixes a lock up of the whole k6.
+- [#3023](https://github.com/grafana/k6/pull/3023) Fixed Trend.Max's support of negative values.
+- [browser#781](https://github.com/grafana/xk6-browser/pull/781) Fixed mapping of `response` object's function from `jSON` to `json`.
+- [browser#779](https://github.com/grafana/xk6-browser/pull/779) Cleared Zombie processes on panic.
+- [browser#834](https://github.com/grafana/xk6-browser/pull/834) Fixed `page.close` so that it closes the current page and not the whole browser context.
+
+## Maintenance and internal improvements
+
+### Improved the per-VU buffer pool [#2879](https://github.com/grafana/k6/pull/2879)
+
+Improved the per-VU buffer pool which should reduce memory usage and improve performance. For some test cases, this change can [reduce memory usage up to 50%](https://github.com/grafana/k6/issues/794#issuecomment-1406522459).
```

Comment:
"Improve performance" is not quite right. [The change actually has a slight negative impact on performance.](https://github.com/grafana/k6/pull/2879#issuecomment-1456164331)

```suggestion
Improved the per-VU buffer pool which should greatly reduce memory usage, at a minor expense of higher CPU usage and lower request throughput. In some cases, this change can [reduce memory usage up to 50%](https://github.com/grafana/k6/issues/794#issuecomment-1406522459).
```